java-plugin/src/main/java/fr/greencodeinitiative/java/checks/ArrayCopyCheck.java

```diff
@@ -41,7 +41,7 @@
         name = "Developpement",
         description = ArrayCopyCheck.MESSAGERULE,
         priority = Priority.MINOR,
-        tags = {"bug"})
+        tags = {"performance", "eco-design", "ecocode"})
```

Comment:
I'm ok with your modifications but please update also json files in `src/main/resources/...` directory. For now, only java plugin has these JSON files to configure rules (tags present on java File aren't used yet).

Reply:
Done